pgmpy/factors/hybrid/FunctionalCPD.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+from pgmpy.factors.base import BaseFactor
+
+
+class FunctionalCPD(BaseFactor):
+    def __init__(self, variable, fn, parents=None):
+        """
+        Parameters:
+        ----------
+        variable: str
+            Name of the variable for which this CPD is defined.
+        fn: callable
```

Comment:
I prefer adding empty lines between each argument. I think it makes it cleaner and easier to read.